src/import/load.js

```diff
@@ -160,7 +177,6 @@ function startsWith(string, searchString) {
 }
 
 load.sanitizeUrl = sanitizeUrl;
-
 load.useXHR = (typeof XMLHttpRequest !== 'undefined');
-
+load.headers = null;
```

Comment:
would it make sense to init it with a {}?  The cost is non existent (this is a global object), but allows users to always treat it as an object and add new values without checking if its set.